static-site/src/sections/individual-group-page.jsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
+import styled from 'styled-components';
+import {
+  HugeSpacer,
+  FlexCenter
+} from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import DatasetSelect from "../components/Datasets/dataset-select";
+import GenericPage from "../layouts/generic-page";
+import { fetchAndParseJSON } from "../util/datasetsHelpers";
+
+function Title({avatarSrc, children}) {
+  if (!children) return null;
+  const AvatarImg = styled.img`
+    width: 140px;
+    margin-right: 20px;
+    object-fit: contain;
+  `;
+  const TitleDiv = styled.div`
+    && {
+      font-weight: 500;
+      font-size: 26px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+  `;
+  return (
+    <div style={{display: "flex", justifyContent: "start", padding: "50px 0px 20px 0px"}}>
+      {avatarSrc ?
+        <AvatarImg alt="avatar" src={avatarSrc}/> :
+        null
+      }
+      <TitleDiv>
+        {children}
+      </TitleDiv>
+    </div>
+  );
+}
+
+function Byline({children}) {
+  if (!children) return null;
+  const Div = styled.div`
+    && {
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 1.428;
+      color: #A9ADB1;
+    }
+  `;
+  return (<Div>{children}</Div>);
+}
+
+function Website({children}) {
```

Comment:
I would use `StyledLink` from `components/Datasets/list-datasets.jsx` instead of this for two reasons:
1. The function signature implies that multiple children can be supplied, but this will be problematic
2. We often want the displayed text to be subtly different to the `href` (e.g. we often leave out the `https://`)